### PR TITLE
Fixed #30013 -- Fixed DatabaseOperations.last_executed_query() with mysqlclient 1.3.14+.

### PR DESCRIPTION
**Pull request by timgraham**
https://code.djangoproject.com/ticket/30013

### CODE REVIEW

File: django/db/backends/mysql/operations.py [hunk 1]
```diff
@@ -141,7 +141,7 @@ def last_executed_query(self, cursor, sql, params):
         # With MySQLdb, cursor objects have an (undocumented) "_last_executed"
         # attribute where the exact query sent to the database is saved.
         # See MySQLdb/cursors.py in the source distribution.
-        query = getattr(cursor, '_last_executed', None)
+        query = getattr(cursor, '_executed', None)
```
Comment (charettes) [inline django/db/backends/mysql/operations.py]: Should we lookup both attributes until we drop support for `mysqlclient<1.3.14`?

Comment (timgraham) [reply django/db/backends/mysql/operations.py]: `_executed` also exists in older versions. I tested with Django's documented minimum, mysqlclient 1.3.7.

File: django/db/backends/mysql/operations.py [hunk 2]
```diff
@@ -141,7 +141,7 @@ def last_executed_query(self, cursor, sql, params):
         # With MySQLdb, cursor objects have an (undocumented) "_last_executed"
```
Comment (pope1ni) [inline django/db/backends/mysql/operations.py]: Don't forget to update the attribute name in the comment!

**Comment by edmorley** [comment]
Makes sense, thank you.
I've filed PyMySQL/mysqlclient-python#303.

**Comment by edmorley** [comment]
Makes sense, thank you.
I've filed PyMySQL/mysqlclient-python#303.

**Comment by edmorley** [comment]
>  I've filed [PyMySQL/mysqlclient-python#303](https://github.com/PyMySQL/mysqlclient-python/issues/303).

This was closed as wontfix.

**Pull request merged**